plotly/grid_objs/grid_objs.py

```diff
@@ -187,3 +254,27 @@ def get_column(self, column_name):
         for column in self._columns:
             if column.name == column_name:
                 return column
+
+    def get_column_reference(self, column_name):
+        """
+        Returns the column reference of given column in the grid by its name.
+
+        Raises an error if the column name is not in the grid. Otherwise,
+        returns the fid:uid pair, which may be the empty string.
+        """
+        col_names = []
+        for column in self._columns:
+            col_names.append(column.name)
```

Comment:
🐄 this is confusing placement of this little code block. Can you move it into the `if column_id is None` block so that:

1. it only needs to be executed on failure
2. it's clear that it's only used in that error message.